config.yaml

```diff
@@ -39,6 +39,19 @@ score_matrix: 'resources/tandem-genotypes/score_matrix.par'
 
 # slivar
 slivar_js: 'resources/slivar/slivar-functions.js'
+# slivar filters
+# gnomad -> frequency in gnomAD v3
+# hprc -> frequency in 40 HPRC samples sequenced with HiFi
+# af -> allele frequency
+# nhomalt -> number of samples with homalt genotype
+# gq -> genotype quality as output by DeepVariant or GLnexus
+max_gnomad_af: 0.01  # for all variants
```

Comment:
I interpret `max` and `min` threshold to be in the accepted range (i.e. that tests would be `X <= max` not `X < max`).

Perhaps change the max counts here to `4` instead of `5` and use `<=` in the slivar expression below.  For the af, I think it's okay to switch from `< 0.01` to `<= 0.01`.

Reply:
Yeah, I had the same thought about `max` but wasn't sure how to handle it for the float values.  Made your recommended changes.